src/index.js

```diff
@@ -56,6 +57,7 @@ async function xxhash() {
       // need to replace our view over it with a new one over the new backing
       // ArrayBuffer.
       memory = new Uint8Array(mem.buffer);
+      temp = new Uint8Array(memory.length);
```

Comment:
I don't think there's a reason to resize this on growth—its sizing should be fixed on initialization to `Math.max(XXH32_STATE_SIZE_BYTES, XXH64_STATE_SIZE_BYTES)`